examples/index.tsx

```diff
@@ -42,6 +42,11 @@ const Root = () => {
     img.src = require('./unicorn.png');
     ref.current?.insertFragment(':unicorn:', img);
   };
+  const customSetValue = () => {
+    ref.current?.setValue(
+      '<span data-rich-mentions=":smile:" class="emojione" style="font-size: 19px; line-height: 19px;">😉</span>    <span data-rich-mentions=":smile:" class="emojione" style="font-size: 19px; line-height: 19px;">😉</span> <img data-rich-mentions=":smile:" src="/images/emojione/2620.png" width="19" height="19" class="emojione vaM" />'
```

Comment:
Maybe it would be clearer to newer contributors if we removed references to our internal classes/styles/images ?

```suggestion
      '<span data-rich-mentions=":smile:">😉</span>    <span data-rich-mentions=":smile:">😉</span> <img data-rich-mentions=":smile:" src="/images/cross"/>'
```

Reply:
I needed to trick it a little as Parcel generate a dynamic path for the image and Cypress doesn't have a way to get it. f19196a